everestpy/everestpy.cpp

```diff
@@ -231,8 +220,7 @@ int initialize(const std::string& prefix, const std::string& config_file, const
         }
 
         if (!everest.connect()) {
-            EVLOG_critical << fmt::format("Cannot connect to MQTT broker at {}:{}", mqtt_server_address,
-                                          mqtt_server_port);
+            EVLOG_critical << fmt::format("Cannot connect to MQTT broker at {}:{}", rs.mqtt_server, rs.mqtt_port);
```

Comment:
concerning naming - choosing mqtt_server_address was my fault, because an address includes the port, if we use mqtt_port, than I would prefer mqtt_host instead of mqtt_server

Reply:
I've no settled on mqtt_broker_host and mqtt_broker_port to make it really clear what is meant by these settings